Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -200,6 +214,63 @@ private void MutateHarvestType(ref HarvestType val, int bits, int totalbits, flo
             val = HarvestType.SelfHarvest;
     }
 
+    private void MutateGases(ref Dictionary<Shared.Atmos.Gas, float> gases, float min, float max, int bits, int totalbits, float mult)
+    {
+        float p = mult * bits / totalbits;
```

Comment:
Try not to name variables as letters unless its obvious what the letter stands for

Reply:
Just tried to follow the style set by the author. All of the other 'Mutate' methods had exact same way of calculating the chance.

Reply:
botany is shitcode dont take it as an example

Reply:
*p* is convention for the probability of an event. This convention has been around for longer than computers and C# existed.